config/crds/all-crds.yaml

```diff
@@ -1542,6 +1542,20 @@ spec:
             config:
               description: Config holds the Enterprise Search configuration.
               type: object
+            configRef:
+              description: 'ConfigRef contains references to Kubernetes Secrets holding
```

Comment:
Maybe we should explain that it is complementary to the `config` field and explain which one takes precedence.